tests/colors/color_picker_component.test.ts

```diff
@@ -110,8 +109,7 @@ describe("Color Picker buttons", () => {
     await simulateClick(".o-color-picker-toggler");
     await simulateClick(".o-gradient");
     const color = "12ef78";
-    setInputValueAndTrigger(".o-custom-input-preview input", color, "input");
```

Comment:
shouldn't all lines with (previously) "input" be converted to `"inputOnly"` ?

Reply:
I don't think so, most of the times when we were only firing "input" on the tests, in a browser both event were fired. We're closer to reality that way. Only one test need to specifically test what happend when when an input is fired an not change